Fenrir/PackDeltaMethods.fs

```diff
@@ -0,0 +1,88 @@
+﻿module Fenrir.DeltaCommands
+
+open System.IO
+open Fenrir.Tools
+open Fenrir.Zlib
+
+let MSBCondition (b: byte): bool =
```

Comment:
I would call it `isMsbSet`, and add a comment that MSB is the Most Significant Bit.